tests/test_redirect.py

```diff
@@ -35,11 +35,11 @@ def test_redirect_url_is_not_recognized_without_include_redirect_urls():
 def test_redirect_url_exception():
     # https://github.com/brave/adblock-rust/blob/b7f29af8c0a0d000201d8d769b6a0b25a9dd4e89/src/blocker.rs#L1314
```

Comment:
@ArniDagur maybe we should change the reference for this test. this behavior has been changed in `0.6.5`.